src/Posts.jsx

```diff
@@ -1,32 +1,235 @@
+const GRAPHQL_ENDPOINT =
+  props.GRAPHQL_ENDPOINT || "https://near-queryapi.api.pagoda.co";
+const LIMIT = 25;
+const sortOption = props.postsOrderOption || "blockHeight"; // following, blockHeight
+let accountsFollowing = props.accountsFollowing;
+const moderatorAccount = props?.moderatorAccount || "bosmod.near";
+
 State.init({
   selectedTab: Storage.privateGet("selectedTab") || "all",
+  posts: [],
+  postsCountLeft: 0,
+  initLoadPosts: false,
+  initLoadPostsAll: false,
 });
 
 const previousSelectedTab = Storage.privateGet("selectedTab");
-
 if (previousSelectedTab && previousSelectedTab !== state.selectedTab) {
   State.update({
     selectedTab: previousSelectedTab,
   });
 }
 
-let accounts = undefined;
-
-if (state.selectedTab === "following" && context.accountId) {
+// TODO port to QueryAPI
```

Comment:
this is intentionally left for later, right?

Reply:
Yes. We fetch the followers using the `Social` API as we do not yet index follower with QueryAPI yet.